homeassistant/components/remote/harmony.py

```diff
@@ -224,12 +224,12 @@ def new_activity(self, activity_info: tuple) -> None:
         self.new_activity(self._client.current_activity)
         await self.hass.async_add_executor_job(self.write_config_file)
 
-    def got_connected(self, _=None):
+    async def got_connected(self, _=None):
```

Comment:
Can the library handle a coroutine instead of a normal callback?

Reply:
Yep, that's the beauty of it. It can handle a future, an event, a coroutine, or a callback. :-) Thus very easy when needed to change it.